### ISSUE TITLE
Could not find or load main class com.thoughtworks.gauge.GaugeRuntime

### ISSUE THREAD

**Issue by dihnatsyeu** [getgauge/Intellij-Plugin#381]
### Expected behavior
Gauge tests should be executed but failed

### Actual behavior
Error: Could not find or load main class com.thoughtworks.gauge.GaugeRuntime
process /usr/bin/java with pid 16626 quit unexpectedly. exit status 1

### Steps to reproduce
Execute gradle gauge with the plugin version 1.7.3
Kotlin was used instead of Java

### Version information

* IntelliJ version and edition(Community/Ultimate):  Ultimate 2019.1
* OS information:MacOs Mojave 10.14.4
* Gauge IntelliJ plugin version: 1.7.3
* Does the project uses any Build tool(Gradle/Maven)? gradle 5.4
```
Gauge version: 1.0.4

Plugins
-------
html-report (4.0.8)
java (0.7.2)
screenshot (0.0.1)

```

**Comment by Apoorva-GA** [comment]
@dihnatsyeu Intellij 2019.1 support is added in gauge-intellij plugin version 0.3.15 which is not yet released. 
So please use the latest nightly version (0.3.15.nightly-2019-04-11). You can fallow [these](https://github.com/getgauge/Intellij-Plugin#installing-nightly) instructions.

**Comment by dihnatsyeu** [comment]
thank you! Unfortunately, it didn't help

**Comment by NivedhaSenthil** [comment]
@dihnatsyeu thanks for reporting I could replicate the issue with the nightly version given for a sample gradle project.. seems to work fine for simple java project.

**Comment by Apoorva-GA** [comment]
@dihnatsyeu Whats the `sourceCompatibility` and `targetCompatibility` set to in your `build.gradle` file?
Please set it to 1.8 and try again. 
I am able to run our sample gradle java project(created using command `gauge init java_gradle`) without any errors.

**Comment by dihnatsyeu** [comment]
> @dihnatsyeu Whats the `sourceCompatibility` and `targetCompatibility` set to in your `build.gradle` file?
> Please set it to 1.8 and try again.
> I am able to run our sample gradle java project(created using command `gauge init java_gradle`) without any errors.

it's not java but kotlin project. 
Source compatability is achieved via 
tasks.withType<KotlinCompile> {
        kotlinOptions.jvmTarget = "1.8"
    }

**Comment by sharath2106** [comment]
Hi everyone 

I face a similar issue while using Gauge with Gradle on Intellij IDEA.

I've my test scripts using Java, Selenium, and Gauge. It was working well earlier (sometimes plugin reinstallations are needed, not sure why) but now, my specs are not recognized even though step definitions are available. 

<img width="542" alt="Screen Shot 2019-05-02 at 3 42 43 PM" src="https://user-images.githubusercontent.com/25933070/57069033-021ec580-6cf1-11e9-9e64-e17cdbf0d01c.png">

When I run the specs using the run icon, I get the following error - 
Error: Could not find or load main class com.thoughtworks.gauge.GaugeRuntime

I use the gauge-gradle plugin 1.7.3

<img width="775" alt="Screen Shot 2019-05-02 at 3 46 04 PM" src="https://user-images.githubusercontent.com/25933070/57069160-6e012e00-6cf1-11e9-9e69-5d38ac9672d6.png">

Please let me know what went wrong now.  If it's an issue, someone pls pick this and fix this asap. 

Happy to provide more details (if needed).

**Comment by sharath2106** [comment]
Adding to the previous comment. I'm currently running the scripts with a tag added to specs as a Gradle task. PFB - 

<img width="1059" alt="Screen Shot 2019-05-02 at 5 36 23 PM" src="https://user-images.githubusercontent.com/25933070/57074251-faffb380-6d00-11e9-9088-9347082da874.png">

**Comment by sharath2106** [comment]
Please find the entire stack trace after trying to run from the Intellij IDEA.

`Testing started at 12:58 ...
/usr/local/bin/gauge run --machine-readable --hide-suggestion --simple-console "/Users/gopishankar/Downloads/Mopar codebase/pricing-ui-tests/specs/UK_Volume_Scenario.spec:9"
Error: Could not find or load main class com.thoughtworks.gauge.GaugeRuntime

process /Users/gopishankar/.asdf/installs/java/oracle-8.141/jre/bin/java with pid 40062 quit unexpectedly. exit status 1

Error ----------------------------------

Failed to start gauge API: Runner with pid 40061 quit unexpectedly(exit status 1).

Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://gitter.im/getgauge/chat

Your Environment Information -----------
	darwin, 1.0.5
	html-report (4.0.8), java (0.7.2), screenshot (0.0.1), spectacle (0.1.3)

Process finished with exit code 1
`

**Comment by negiDharmendra** [comment]
@sharath2106 Are you able to run `Gauge specs` from the command line.
Which version of IntelliJ and Gauge-IntelliJ plugin you are using.

**Comment by shubhamsc** [comment]
@sharath2106 I can see this issue with IntelliJ 2018.3 and gauge-IntelliJ-plugin v0.3.14. 
With the latest gauge-IntelliJ-plugin, I'm not seeing any issue.
Can you update the plugin to the latest released v0.3.15?
Let's know if you are still facing the same issue with the latest gauge plugin.

**Comment by JanisSharma** [comment]
I got this error because I was trying to run

javac HelloWorld.java && java HelloWorld.class
when I should have removed .class:

javac HelloWorld.java && java HelloWorld

Get Full Code At [Here](https://icetutor.com/question/error-could-not-find-or-load-main-class/)
